README.md

```diff
@@ -1,4 +1,7 @@
-# bpftrace
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/bpftrace/website/09ad78b583af653a0f373e858957864c286554c0/static/img/bpftrace_Full_Logo-White.svg"/>
```

Comment:
Meh, it's not a big image. Do we really have concerns with adding it to this repo?

Reply:
Yeah we've already got an images directory: https://github.com/bpftrace/bpftrace/tree/master/images

Reply:
Huh, I didn't even notice that. Weird.